docs/lang-guides/node.md

```diff
@@ -0,0 +1,4 @@
+# NodeJS
+
+For a complete Earthfile example on how to use NodeJS in Earthly, visit the [js example directory on GitHub](https://github.com/earthly/earthly/tree/main/examples/js) or the [typescript-node example directory on GitHub](https://github.com/earthly/earthly/tree/main/examples/typescript-node).
+
```

Comment:
nit: trailing newline, which will cause issues with the `+lint-newline-ending` test, which unfortunately wasn't triggered on this PR.